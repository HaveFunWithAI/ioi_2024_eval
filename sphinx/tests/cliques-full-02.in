250 2417
7 132 71 42 132 87 71 71 50 66 66 198 198 212 212 18 198 87 212 219 87 87 66 219 219 42 66 132 132 198 18 71 87 50 132 50 50 66 50 66 7 71 66 119 7 219 132 71 132 66 132 42 198 66 132 42 119 219 42 66 50 87 18 119 66 7 18 18 132 132 87 18 87 87 198 50 66 87 132 198 87 212 66 7 198 66 71 66 132 219 42 7 119 212 212 18 219 212 119 132 87 198 119 219 219 42 7 119 219 50 18 7 132 212 66 50 212 7 71 71 119 42 212 71 66 71 42 71 42 219 119 87 212 219 212 219 7 119 87 7 50 119 198 71 212 198 7 87 219 87 18 18 50 66 132 87 7 87 42 198 198 18 50 119 119 87 87 132 18 119 42 119 66 71 7 7 87 198 119 198 18 7 71 18 212 18 42 66 42 18 7 42 50 198 7 71 132 7 18 212 18 66 132 7 219 66 42 7 42 132 7 212 66 87 219 71 66 87 7 18 132 42 18 132 50 7 132 42 219 71 42 7 18 50 50 87 119 50 132 219 50 198 7 132 18 198 66 42 18 18
36 241
54 224
60 244
117 238
187 210
68 129
54 74
4 147
148 215
227 237
65 178
118 125
105 237
118 235
88 98
108 110
111 239
0 97
49 242
76 108
137 147
47 210
73 229
56 180
167 239
30 56
88 173
114 183
93 181
128 145
9 113
182 249
102 242
64 211
17 124
116 158
10 220
111 160
163 165
65 177
153 217
95 181
83 245
130 142
205 212
1 191
3 95
158 172
48 75
89 172
146 163
119 156
124 137
134 225
57 205
35 86
6 112
17 147
98 131
62 170
115 212
33 153
85 128
45 206
18 238
99 110
57 102
183 191
25 60
63 137
29 85
116 192
42 71
208 224
4 17
11 77
211 231
91 168
38 107
33 231
111 221
70 111
51 178
45 47
149 177
220 232
24 206
19 180
24 31
75 204
28 112
155 180
138 181
218 239
10 95
40 105
143 165
120 187
136 193
63 157
9 238
49 249
144 184
77 130
37 213
64 164
75 94
27 217
103 115
18 154
106 195
150 178
45 230
130 172
42 96
3 236
152 164
151 162
4 51
15 26
36 63
1 171
42 169
5 247
156 185
159 163
105 227
148 242
118 177
52 206
43 151
209 226
119 185
3 171
148 212
82 179
167 196
143 159
179 228
67 247
164 248
8 68
21 145
89 130
137 178
4 137
219 222
20 92
58 192
24 187
112 222
64 228
87 138
59 126
89 234
85 243
7 89
144 163
40 64
127 145
137 150
88 246
187 206
7 234
83 141
49 119
239 246
24 224
113 154
55 218
9 41
64 179
57 156
14 176
69 208
154 155
144 244
131 239
145 189
48 133
152 179
7 158
31 120
71 127
32 223
11 234
22 64
34 61
53 139
157 170
181 232
79 219
80 240
40 230
80 81
37 49
54 110
1 75
118 241
125 178
107 244
25 244
97 249
91 106
131 167
26 50
58 142
99 208
125 150
66 93
71 145
95 236
193 200
102 148
114 161
153 231
62 65
76 226
77 80
112 133
211 228
27 230
110 179
109 227
128 203
21 194
95 113
36 51
217 228
140 200
2 171
83 181
150 241
134 174
33 82
83 93
6 197
77 192
50 152
77 81
84 133
87 230
13 194
44 233
211 230
62 235
115 242
51 65
51 63
57 207
32 200
47 54
164 227
37 115
40 227
69 99
21 189
6 84
58 234
87 141
12 201
42 243
147 150
43 123
13 96
160 239
7 139
18 117
140 195
35 200
24 54
135 176
59 166
114 146
197 222
227 248
84 112
79 94
189 229
49 115
87 95
10 50
25 38
132 223
24 108
13 127
30 117
22 196
151 166
125 147
63 177
213 249
36 170
54 69
49 185
33 152
46 221
31 108
44 225
105 248
88 221
171 191
135 167
77 158
34 119
31 52
59 117
127 128
10 232
129 201
139 158
90 220
152 227
28 122
85 92
11 81
149 170
27 228
33 105
44 94
29 189
73 194
82 152
30 113
72 245
109 152
105 217
88 196
118 150
157 178
27 164
33 109
105 230
56 123
38 183
173 246
152 217
102 215
62 137
139 234
7 100
48 84
1 38
56 166
167 169
100 158
84 122
75 174
85 121
163 190
156 212
70 160
22 153
197 219
134 219
86 193
37 102
75 79
18 30
111 246
211 248
19 154
26 95
227 230
100 175
46 98
20 96
50 181
24 210
121 169
80 139
28 233
103 213
46 173
104 132
134 233
156 205
26 93
113 238
116 188
43 180
96 145
15 141
102 207
132 186
52 120
66 220
39 176
94 112
183 190
95 138
84 204
161 190
73 127
82 153
30 180
47 69
0 132
153 237
104 200
14 46
153 164
214 236
151 154
155 238
181 245
117 126
55 135
12 199
35 91
82 164
76 206
6 94
2 163
32 35
52 108
133 174
123 154
31 206
56 59
25 184
21 243
164 230
175 234
85 169
28 174
7 188
217 230
122 133
146 171
27 33
36 147
84 225
76 208
104 195
95 214
50 245
9 126
28 48
40 228
133 197
23 134
105 179
173 221
6 28
127 189
156 215
2 144
49 205
144 190
60 165
25 171
9 166
225 233
123 198
44 133
6 233
87 90
107 165
31 187
197 225
69 120
196 218
78 167
112 197
77 234
122 242
164 228
47 226
40 211
53 89
20 121
54 187
156 242
102 183
128 234
27 64
89 139
81 139
2 183
74 206
82 248
92 121
39 111
68 201
152 248
69 187
25 107
182 207
84 233
96 169
122 204
33 179
187 208
30 166
35 202
115 156
23 112
37 212
10 181
13 71
18 198
27 109
214 232
127 169
88 131
107 161
84 134
7 80
94 204
33 126
93 236
58 116
108 120
211 227
37 215
10 79
22 237
130 175
40 237
72 141
124 235
181 220
98 221
38 191
77 240
95 245
210 226
46 196
168 216
85 203
22 211
21 127
182 242
137 157
94 222
196 239
1 163
104 186
156 213
3 15
61 148
159 190
125 137
57 212
162 238
72 181
231 248
4 150
78 98
166 238
35 223
163 183
126 180
175 240
33 227
132 195
101 129
117 166
3 245
165 184
134 197
17 137
69 226
180 198
24 99
98 111
15 181
48 225
45 52
140 202
2 146
96 127
66 214
136 186
38 184
232 245
134 204
35 97
104 216
72 90
42 121
15 214
40 109
121 229
46 70
13 29
23 44
16 32
143 203
72 93
5 169
25 163
24 110
59 180
22 179
11 142
47 120
194 243
4 124
106 140
24 69
81 188
36 62
60 159
34 242
6 225
70 246
97 200
61 205
137 235
11 172
215 249
73 96
143 163
85 229
43 166
70 135
151 198
195 202
38 171
172 234
106 186
135 246
110 226
94 225
13 121
44 222
85 127
207 249
156 207
16 104
130 222
207 215
27 211
90 214
103 249
56 198
18 41
177 241
14 111
75 112
28 204
20 203
34 212
164 179
160 221
120 193
208 209
61 119
174 219
89 158
32 97
108 224
39 239
204 225
167 173
99 210
47 206
203 243
94 217
52 74
108 209
128 229
107 190
7 77
83 220
13 169
148 213
90 95
71 243
124 241
30 198
81 240
20 145
119 242
184 190
18 123
47 76
83 232
62 241
28 84
169 189
54 210
14 218
16 223
64 227
145 169
9 198
164 237
91 223
45 208
147 157
86 97
83 87
228 237
0 193
94 134
106 136
12 68
210 224
179 217
48 219
29 73
123 238
79 134
114 244
92 203
43 198
46 239
64 82
24 76
13 20
3 93
211 237
108 187
107 183
16 202
107 163
123 151
152 153
41 59
119 205
228 230
16 186
70 131
18 166
122 134
0 200
103 119
129 199
80 175
92 229
153 228
118 124
14 221
76 209
71 128
18 19
107 114
81 172
104 140
82 227
25 114
167 221
91 140
32 140
199 201
116 139
97 140
20 229
49 213
22 152
132 140
57 213
46 78
193 205
29 194
23 197
37 156
8 199
53 80
113 126
145 203
62 147
170 178
207 212
119 215
146 184
51 157
141 236
13 21
20 29
144 183
79 138
147 170
67 129
178 191
185 249
117 162
96 189
159 183
71 203
152 230
8 201
9 180
189 203
66 87
82 230
66 189
212 213
34 249
43 59
206 210
14 39
24 208
117 198
88 160
113 155
186 195
89 175
147 235
82 211
71 169
19 166
23 84
106 223
161 183
205 213
123 180
85 145
26 181
146 159
39 98
194 229
76 210
158 240
0 35
173 196
122 174
40 248
117 155
74 110
4 229
109 237
98 218
144 159
103 242
78 221
14 246
42 128
23 204
123 155
144 171
50 138
61 249
96 194
35 104
23 94
20 21
186 200
113 123
156 182
78 131
82 109
91 200
106 216
92 145
147 149
118 154
142 172
17 118
149 157
94 197
40 231
43 56
133 225
193 195
114 165
45 209
82 105
126 162
105 164
73 189
55 173
185 207
118 147
2 165
9 155
211 217
23 48
30 59
5 201
124 147
153 227
103 185
70 167
1 146
28 219
137 241
41 43
107 146
16 195
11 240
102 182
138 236
168 223
144 161
15 87
71 189
0 136
47 99
124 170
43 126
107 184
120 206
121 243
136 200
6 77
125 177
71 73
37 103
72 95
182 233
80 158
60 114
67 68
44 219
32 132
11 192
99 108
166 198
143 191
173 239
222 233
128 189
187 226
19 43
17 65
70 239
58 158
98 176
57 182
34 102
181 214
157 175
126 166
22 231
96 128
43 238
44 84
202 223
193 202
94 219
30 155
155 198
36 235
98 246
4 62
19 198
32 106
78 88
55 88
41 162
16 136
173 218
61 212
53 58
52 209
83 138
114 144
72 236
125 241
23 122
161 163
15 236
61 156
119 148
109 230
153 248
17 62
109 228
83 236
29 42
136 202
9 59
213 215
46 135
97 186
156 249
3 72
18 126
79 133
62 177
71 92
122 197
135 186
24 209
63 178
31 76
70 78
103 212
2 190
70 88
218 221
88 218
79 84
90 138
78 149
91 195
69 108
31 172
39 46
191 244
1 190
1 144
112 233
99 178
36 157
69 110
13 73
9 19
137 177
40 153
59 113
85 194
90 236
110 208
2 107
80 89
55 221
129 247
9 154
41 238
68 247
42 145
20 194
165 190
15 127
36 118
17 235
90 93
0 104
31 99
35 186
179 230
131 221
34 148
185 205
13 85
11 188
105 211
110 224
58 172
116 142
71 85
121 145
176 218
152 231
51 147
133 204
14 55
21 29
16 180
148 249
101 201
194 203
37 207
41 166
65 170
14 239
135 160
193 223
15 220
33 64
146 244
66 90
24 45
0 16
15 139
74 120
6 133
146 161
159 184
145 243
125 235
59 155
34 207
11 175
128 243
86 168
123 166
54 108
157 241
7 53
145 194
31 54
108 210
36 150
77 142
44 59
45 226
96 203
63 149
100 240
14 135
120 208
110 209
4 177
37 249
136 216
102 156
179 248
130 188
90 141
44 122
42 194
39 218
88 176
119 249
7 130
15 93
10 138
59 198
65 124
28 222
160 218
92 189
1 114
22 33
38 163
147 178
49 156
183 244
103 156
39 160
80 205
18 155
237 248
51 235
134 222
130 158
48 222
162 180
73 128
127 243
64 237
14 98
40 152
92 243
111 218
40 179
120 210
57 119
74 208
89 116
91 202
147 241
118 170
8 101
39 78
123 126
2 244
86 91
167 176
11 100
7 192
22 248
58 139
80 100
130 234
39 135
61 215
65 150
138 245
38 190
65 118
55 70
98 167
122 138
17 63
119 207
76 120
99 224
42 189
91 186
22 109
182 185
3 214
52 224
36 45
127 203
98 160
184 191
71 121
206 226
13 92
58 81
25 146
31 209
19 162
10 83
57 185
189 243
124 150
220 236
93 138
125 149
75 84
4 65
126 151
136 168
51 62
69 210
105 152
112 134
40 164
119 182
95 141
123 162
104 235
113 198
208 221
222 225
90 245
185 242
106 168
63 125
22 105
100 116
82 237
112 225
86 140
10 87
182 205
28 225
152 211
0 186
39 55
17 178
55 239
204 222
164 231
38 165
75 233
32 46
108 208
150 170
72 232
55 196
169 203
111 173
120 224
9 162
196 221
142 192
55 160
6 44
71 229
201 247
176 246
15 138
4 170
19 155
66 141
103 246
10 236
177 178
32 193
56 151
94 228
110 206
51 124
73 85
53 142
60 161
109 211
192 240
145 170
74 209
16 106
26 138
150 157
51 150
26 236
5 67
82 228
148 205
81 116
183 184
60 171
16 97
131 246
200 216
190 244
113 180
0 195
131 160
102 185
159 161
1 25
115 215
14 88
158 188
60 191
114 190
71 96
32 168
66 181
65 241
29 96
168 186
48 79
81 175
81 89
48 122
23 75
3 232
26 232
25 190
163 244
186 193
2 143
90 232
3 181
112 219
51 125
57 215
65 235
44 75
6 48
136 223
107 143
228 231
101 247
92 194
84 222
61 102
34 213
78 233
56 238
124 157
41 117
35 168
48 112
34 57
105 109
35 216
17 241
21 85
97 202
110 210
236 245
20 243
1 184
76 99
142 188
73 169
127 229
163 191
4 36
36 149
17 177
103 148
47 52
116 130
108 206
74 187
45 224
15 90
71 194
161 244
149 178
32 222
99 131
209 224
1 165
29 121
21 128
80 116
31 69
67 101
171 244
79 233
0 106
16 200
86 200
17 170
28 59
135 173
64 153
66 232
25 191
0 140
19 56
37 119
31 208
18 151
95 220
23 174
15 50
15 83
160 176
31 45
87 93
20 128
86 223
21 92
62 118
31 210
11 158
147 177
158 234
74 76
65 149
0 168
34 215
4 125
29 127
207 242
151 180
25 183
124 177
152 228
209 210
128 194
122 222
1 161
21 76
101 199
87 220
86 186
60 143
63 150
135 196
60 183
94 122
165 183
29 169
86 136
97 168
6 134
113 166
162 198
115 148
7 11
32 91
3 26
64 230
99 206
139 142
11 58
9 117
84 174
18 59
192 234
32 230
186 223
25 159
0 216
97 195
102 249
0 125
34 37
20 189
161 171
104 168
51 149
149 150
56 155
47 208
86 104
34 156
135 221
139 188
97 216
27 152
117 151
12 101
66 83
44 79
52 99
150 235
10 141
12 67
50 95
198 238
64 152
102 212
117 154
100 139
112 174
135 239
138 141
154 166
27 237
207 213
231 237
88 239
60 190
38 60
14 78
167 218
37 185
78 246
199 247
92 96
53 172
57 249
45 187
69 224
57 198
3 66
78 218
50 220
92 128
24 74
13 145
5 68
23 28
80 142
133 222
97 193
70 196
217 248
53 240
46 88
0 223
25 144
44 197
23 79
77 172
114 159
32 86
113 117
62 157
55 78
158 192
204 233
19 89
70 221
52 226
157 190
72 87
45 210
22 230
54 120
22 217
131 196
121 194
15 72
54 226
91 193
93 245
55 246
176 239
202 235
110 187
97 223
43 162
61 242
81 158
75 122
97 106
31 47
153 211
18 43
68 199
110 198
6 222
115 207
110 120
234 240
219 225
153 223
41 126
227 231
78 239
116 240
128 169
14 167
13 189
50 66
70 218
39 124
37 182
35 140
20 169
212 242
0 91
59 162
44 204
106 132
39 88
120 209
20 127
174 197
181 236
15 232
65 147
29 128
172 240
66 95
41 113
230 248
22 164
78 135
6 122
165 171
19 151
88 167
50 214
9 43
10 26
83 90
42 203
93 214
205 215
44 134
93 141
164 217
53 192
176 221
47 187
1 107
66 138
133 233
77 188
55 131
176 196
56 126
218 247
23 133
13 243
57 148
45 99
70 177
63 147
96 243
33 40
58 130
44 174
13 42
185 212
53 175
100 142
9 30
150 177
26 90
97 104
86 132
98 135
62 125
48 204
27 231
28 94
78 160
46 167
10 72
51 137
100 188
141 245
188 240
112 122
7 81
114 171
49 182
15 245
11 80
30 162
38 143
200 202
42 85
3 83
84 197
186 202
30 43
105 231
4 235
198 249
160 196
215 242
107 171
93 232
53 116
7 142
52 208
81 142
159 171
92 169
149 241
2 161
144 146
212 215
133 219
165 244
86 202
73 145
18 180
59 154
56 154
7 58
3 87
132 200
27 153
62 178
154 180
0 32
15 81
49 103
104 193
4 118
50 236
43 155
76 110
87 236
36 177
18 113
122 233
85 96
142 234
59 238
109 217
22 228
46 160
84 219
138 232
75 197
143 184
118 137
208 226
30 41
168 202
41 56
36 124
137 149
206 224
6 79
161 165
48 94
50 90
75 219
159 165
13 203
9 123
232 236
11 89
19 238
28 134
99 226
30 123
153 230
67 180
124 149
10 90
51 170
13 128
80 188
170 177
30 151
8 247
73 92
121 189
97 132
126 154
21 229
39 246
15 95
217 237
79 174
30 154
14 160
179 211
4 63
31 224
127 194
31 226
93 220
187 209
11 53
27 248
172 175
140 168
16 140
59 151
20 42
33 211
30 238
25 165
153 179
86 216
66 236
137 170
154 162
16 35
115 119
48 134
100 130
146 191
37 148
109 136
151 238
72 83
24 120
37 61
142 158
197 204
69 206
203 229
45 74
60 146
27 227
58 188
87 214
39 196
6 174
5 199
62 150
39 221
77 100
214 220
109 231
144 165
87 181
104 106
33 230
154 198
73 121
10 15
135 218
88 111
104 202
27 179
23 222
3 138
148 185
115 249
1 2
86 195
10 245
148 207
21 121
70 173
98 196
139 172
64 217
35 193
50 87
19 59
44 48
51 177
179 231
190 191
54 206
195 223
78 176
139 192
90 96
61 182
64 231
119 213
125 170
77 139
117 123
168 195
46 131
31 74
94 233
144 191
195 200
22 246
2 191
51 118
124 178
79 190
205 242
34 185
117 180
159 244
29 229
26 83
11 130
94 133
86 106
19 123
49 212
49 207
6 219
84 94
10 214
126 198
18 56
64 109
229 243
102 119
89 192
221 246
53 100
3 10
61 213
49 102
23 233
148 156
195 216
35 90
109 179
119 212
14 131
114 143
49 148
142 240
103 215
165 191
157 177
182 213
89 100
116 234
41 155
59 123
217 231
82 231
146 190
6 23
8 67
111 131
163 171
58 77
63 235
56 113
188 234
41 180
32 195
65 206
171 183
205 249
178 241
4 149
68 101
194 205
48 197
114 191
74 210
75 222
212 249
74 99
170 241
107 191
16 168
185 213
132 168
26 66
220 245
155 166
143 161
29 92
19 30
57 103
1 159
16 216
42 92
197 233
140 216
87 245
63 241
54 208
130 192
114 184
104 136
9 56
43 113
5 8
143 171
9 18
73 203
106 200
143 144
52 210
145 229
35 132
46 111
78 196
69 76
79 222
63 65
4 241
204 219
140 193
111 167
213 242
112 204
12 129
60 163
96 229
7 175
34 205
46 55
77 116
115 213
38 146
115 205
52 69
4 178
22 82
25 161
24 52
200 223
2 60
19 117
16 91
161 191
102 213
26 214
146 183
74 108
55 98
39 131
32 136
54 76
97 136
122 219
159 191
77 89
46 218
17 150
65 125
95 232
136 140
174 233
132 216
99 187
26 72
131 135
23 219
22 40
53 81
155 162
36 125
67 201
115 182
10 66
12 247
42 127
111 135
33 217
172 188
118 157
63 118
55 111
126 155
52 76
3 220
106 193
39 70
230 237
180 238
42 229
151 155
168 193
76 224
89 240
43 117
53 188
114 163
2 184
169 194
3 50
139 175
91 216
6 204
143 190
77 175
36 178
3 90
2 43
174 225
216 223
17 157
94 174
35 195
19 113
47 224
11 116
27 82
34 103
169 243
29 203
221 239
72 220
34 49
92 127
25 126
178 235
166 180
41 123
80 234
102 205
130 139
26 220
83 95
50 83
196 246
228 248
22 227
206 208
119 143
75 225
118 178
1 244
5 12
102 103
60 184
179 227
5 101
48 91
81 130
217 227
52 54
58 100
139 186
38 114
89 188
140 186
47 110
2 114
13 229
53 158
82 217
36 65
65 137
171 190
2 38
111 196
88 135
182 215
15 144
90 181
104 223
122 225
163 184
58 80
143 244
118 149
136 195
21 169
126 238
72 214
49 61
52 187
32 104
91 104
2 25
121 128
58 240
25 143
55 167
21 71
20 71
57 242
106 202
79 122
16 132
70 176
75 133
32 186
50 232
143 183
56 117
164 211
32 216
47 74
79 197
175 192
14 70
100 234
28 75
70 98
17 149
31 110
175 188
57 115
125 157
28 44
19 41
107 144
138 214
28 79
169 229
121 127
179 237
66 72
15 66
65 157
41 198
141 181
50 141
160 246
160 167
74 224
37 205
19 126
139 240
58 175
17 125
93 95
56 162
53 234
24 47
26 141
99 120
61 185
67 199
33 237
1 60
76 187
7 144
63 170
167 246
72 138
42 73
38 47
81 234
8 107
43 154
146 165
27 105
27 40
52 110
35 208
21 73
74 226
62 149
177 235
91 132
0 202
53 130
242 249
105 153
8 129
57 61
78 111
69 74
205 207
107 159
158 175
26 87
87 232
66 245
48 233
28 133
212 220
38 244
134 170
182 212
33 228
17 51
85 189
10 93
78 173
46 246
16 193
32 202
142 175
63 124
208 210
173 176
48 174
193 216
189 194
161 184
55 176
184 244
19 136
186 216
45 54
230 231
8 12
62 63
132 193
33 248
75 134
38 144
91 136
1 183
58 145
79 225
131 218
121 203
174 204
39 173
171 184
133 134
91 97
18 162
51 241
98 173
227 228
46 176
0 86
7 240
108 211
37 57
49 215
50 93
188 192
45 108
7 172
39 167
168 200
206 209
26 245
162 166
107 196
79 204
44 112
138 220
115 185
109 248
98 239
14 196
131 173
20 85
34 115
40 217
3 141
100 192
116 175
143 146
170 235
102 115
103 205
60 107
45 120
80 172
1 143
24 226
160 173
157 235
116 172
54 99
64 105
61 115
120 226
81 100
57 118
47 209
61 103
61 207
80 192
34 182
7 116
124 125
30 126
83 214
80 130
105 228
148 182
218 246
53 77
54 209
214 245
64 248
130 240
41 151
50 210
41 154
100 172
103 207
47 108
22 27
96 121
140 223
81 192
103 182
29 145
219 233
172 192
62 124
132 202
58 89
35 136
108 226
21 203
11 139
109 164
4 157
16 86
113 151
111 176
36 137
73 243
35 106
6 75
131 176
224 226
60 162
174 222
152 237
60 144
154 238
50 72
33 164
5 129
37 242
20 73
149 235
38 159
45 76
202 216
89 142
17 36
185 215
45 110
141 220
141 214
79 110
69 209
21 96
40 82
28 197
9 151
109 153
2 159
235 241
49 57
113 162
29 243
23 225
14 173
29 71
21 42
141 232
187 224
132 136
99 209
79 112
45 69
38 161
