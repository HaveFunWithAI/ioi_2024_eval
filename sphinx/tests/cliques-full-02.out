250 2327
7 132 71 42 132 87 71 71 50 66 66 198 198 212 212 18 198 87 212 219 87 87 66 219 219 42 66 132 132 198 18 71 87 50 132 50 50 66 50 66 7 71 66 119 7 219 132 71 132 66 132 42 198 66 132 42 119 219 42 66 50 87 18 119 66 7 18 18 132 132 87 18 87 87 198 50 66 87 132 198 87 212 66 7 198 66 71 66 132 219 42 7 119 212 212 18 219 212 119 132 87 198 119 219 219 42 7 119 219 50 18 7 132 212 66 50 212 7 71 71 119 42 212 71 66 71 42 71 42 219 119 87 212 219 212 219 7 119 87 7 50 119 198 71 212 198 7 87 219 87 18 18 50 66 132 87 7 87 42 198 198 18 50 119 119 87 87 132 18 119 42 119 66 71 7 7 87 198 119 198 18 7 71 18 212 18 42 66 42 18 7 42 50 198 7 71 132 7 18 212 18 66 132 7 219 66 42 7 42 132 7 212 66 87 219 71 66 87 7 18 132 42 18 132 50 7 132 42 219 71 42 7 18 50 50 87 119 50 132 219 50 198 7 132 18 198 66 42 18 18
